dstu3/src/main/java/org/opencds/cqf/dstu3/evaluation/ProviderFactory.java

```diff
@@ -54,13 +58,13 @@ public DataProvider createDataProvider(String model, String version, Terminology
     }
 
     public TerminologyProvider createTerminologyProvider(String model, String version, String url, String user, String pass) {
-        if (url != null && url.contains("apelon.com")) {
-            return new Dstu3ApelonFhirTerminologyProvider(this.fhirContext)
-            .withBasicAuth(user, pass).setEndpoint(url, false);
+        if(url != null && !url.isEmpty()){
+            IGenericClient client = ClientHelper.getClient("dstu3", url, user, pass);
```

Comment:
For now, just pass in FhirContext.forDstu3().

The reason is that the cqf-ruler already builds certain fhir contexts on startup and we can just figure out how to pass them in.